packages/hurumap-core/jest.config.js

```diff
@@ -8,6 +8,8 @@ module.exports = {
     ...moduleNameMapper,
     // Handle module aliases
     "^@/commons-ui/core/(.*)$": "<rootDir>/../commons-ui-core/src/$1",
+    "^@/commons-ui/next/(.*)$": "<rootDir>/../commons-ui-next/src/$1",
+    "^@/commons-ui/payload/(.*)$": "<rootDir>/../commons-ui-payload/src/$1",
```

Comment:
```suggestion
```
Why would `@hurumap/core` touch on anything payload related?

Reply:
@kilemensi This is because we are using `RichText` from `payload`

Reply:
To clarify @kelvinkipruto, as you can see from the dependencies: https://github.com/CodeForAfrica/ui/blob/cd2eb9ad09501095947b7b2ea5005083522d234a/packages/hurumap-core/package.json#L61-L75

1. `@hurumap/core` only depends on `react`, `@mui/material` (i.e. `@commons-ui/core`) plus charting libraries e.g. `vega`.
2. there shouldn't be any dependencies on `next`, `payload`, etc.

Reply:
Thanks, @kilemensi Once I saw this, everything became clear. It's now ready for a final review.